monitoring/base/rhods-rules.yaml

```diff
@@ -16,5 +16,9 @@ spec:
       expr: count(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}==1)
     - record: cluster:usage:consumption:rhods:active_users
       expr: count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}[1h]))
+    - record: cluster:usage:consumption:cpu:requests:runtime
```

Comment:
@DharmitD this metric name needs to have "rhods" in it somewhere. I'd recommend:
`cluster:usage:consumption:rhods:cpu_requests_runtime`

Reply:
Done, updated the name to have "rhods".